bundles/Planning/Resources/assets/planning.js

```diff
@@ -144,13 +144,11 @@ require('./planning.css');
             getStatusLevelClass: function (user, sprint) {
                 let remainingEstimateUser = this.getRemainingEstimatUser(user, sprint);
 
-                if (remainingEstimateUser > 35) {
+                if (Number(remainingEstimateUser) === 60) {
```

Comment:
Why Number() here?

if it is needed, it should be applied at the 
"let remainingEstimateUser = this.getRemainingEstimatUser(user, sprint);"
line above instead

Reply:
Otherwise the comparison fails as the remainingEstimateUser is not a number, and that makes comparison by === fail. I will move it.